vyper/__init__.py

```diff
@@ -21,3 +21,6 @@
     __version__ = _version(__name__)
 except PackageNotFoundError:
     from vyper.version import version as __version__
+
+# pep440 version with commit hash
+__long_version__ = f"{__version__}+commit.{__commit__}"
```

Comment:
`setuptools-scm` can do this now as of v8.0 btw
https://github.com/pypa/setuptools_scm/pull/854/files#diff-65e9e92414548be02d6e27fcb82e0a5b4192ce9951f6b2b29544a31bb46dbb97R25

Reply:
i'm skeptical, i remember encountering a lot of issues with setuptools_scm around this

Reply:
It is a very good tool, and now works well with pyproject.toml only workflow